pom.xml

```diff
@@ -71,8 +71,10 @@
         <opencsv.version>5.2</opencsv.version>
         <farao.bom.version>1.1.0</farao.bom.version>
         <farao.core.version>3.3.3</farao.core.version>
+        <farao.rte.core.version>2.5.0</farao.rte.core.version>
```

Comment:
These variables don't have to be overriden, it is better to only change version of farao-parent-ws from 1.0.10 to 1.1.0